src/docs/firmware/manually-flashing.md

```diff
@@ -4,6 +4,28 @@
 Flashing your own firmware has the potential to brick your device. Do not do this unless you are sure you know what you're doing/have a way to recover from a bad flash. Some level of knowledge with using the Linux command line is required.
 :::
 
+## Persisting the board's Vital Product Data (VPD)
+
+The firmware in all ChromeOS devices contains a section (RO_VPD) which stores board-specific data, like the serial number, localization settings, and on many devices which have an Ethernet port, the LAN MAC address as well. When flashing via the Firmware Utility Script, the script will automatically extract this from the running firmware and inject it into the firmware to be flashed, so the device serial, LAN MAC address, etc are all maintained. Without this, the device will use a default/generic LAN MAC address set by coreboot. While not ideal, this is only really an issue if two or more of the same device are on the same LAN segment (or you're statically assigning IP addresses based on MAC). But for completeness, if flashing the UEFI firmware or shellball ROM, we'll extract the VPD (either from the board itself or a backup made by the script) and inject it into the firmware to be flashed.
+
+::: tip
+You don't need to do this if flashing a stock firmware backup created by the Firmware Utility Script; that image already contains the VPD.
+:::
+
+1. For both the options below, we'll need to use the cbfstool (coreboot filesystem) binary, so let's download/extract that:
+    * `wget https://mrchromebox.tech/files/util/cbfstool.tar.gz && tar -zxf cbfstool.tar.gz`
+    * Option 1: Extract VPD from the firmware on device
+        * `sudo flashrom -p ch341a_spi -r badflash.rom`
```

Comment:
missed this the first time, should be internal

Reply:
fixed